Dockerfile-build

```diff
@@ -0,0 +1,22 @@
+FROM node:0.10.45
```

Comment:
There's an existing Dockerfile that's used for fxa self-hosting - I think we'll look at tackling the self-hosting/local dev after we get it working in prod